<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopPatchesWidget</class>
 <widget class="QWidget" name="TopPatchesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Top Patches</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,0,0,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Target patch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="gfxTargetPatch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Clusters:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtClusters">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Best patches:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtNumberOfPatches">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnCompute">
         <property name="text">
          <string>Compute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tblviewTopPatches"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
